### PR TITLE
Radio tapes

### PR DESCRIPTION
**Pull request by PhuzzySA**
## About the PR 

I've got three more tapes for the radio station, though only two will be getting added to the standard bin in the satellite or the comms offices. One of them is getting hidden in the loot crates for miners and telescience to find. (hopefully)

Thanks again to members of the Black Pants Legion for lending their voices, including Tex himself, who's got two tapes in the mix. You'll have to go and find one of them yourself. 

## Why's this needed? 

Increasing the variety of voices on the radio station, rewarding crewmates for hunting down elusive loot.

## Changelog

```
(u)Phuzzy:
(*)Adding three new radio tapes to the station! One of them is a secret though, and you'll have to go and find it yourself.
```

### CODE REVIEW

File: code/__secret_public.dme [hunk 1]
```diff
@@ -14,3 +14,14 @@
...
 #endif
+// BEGIN_INTERNALS
+// END_INTERNALS
+// BEGIN_FILE_DIR
+#define FILE_DIR .
+// END_FILE_DIR
+// BEGIN_PREFERENCES
+// END_PREFERENCES
+// BEGIN_INCLUDE
+#include "_placeholder.dm"
```
Comment (ZeWaka) [inline code/__secret_public.dme:25]: gotta remove all this junk

Comment (ZeWaka) [reply code/__secret_public.dme:25]: this is why you don't open up the secret.dme as a dme

File: code/modules/telescience/radiostation.dm [hunk 2]
```diff
@@ -600,13 +600,15 @@
 	name = "compact tape - 'Discount Dan's Quik Noodles'"
 	audio = "sound/radio_station/quik_noodles.ogg"
 	name_of_thing = "Discount Dan's Quik Noodles"
+	desc = {"A small audio tape. Though, it looks too big to fit in an audio log.<br>
+	The music is "Palast Rock by Stefan Kartenberg (CC BY-NO 3.0)"}
```
Comment (ZeWaka) [inline code/modules/telescience/radiostation.dm:604]:
```suggestion
	The music is "Palast Rock by Stefan Kartenberg (CC BY-NC 3.0)"}
```

File: code/modules/telescience/radiostation.dm [hunk 3]
```diff
@@ -626,7 +629,31 @@
 	name_of_thing = "Cloning Public Service Announcement"
 	desc = {"A small audio tape. Though, it looks to big to fit in an audio log.<br>
 	Voiceover by Cenith of the Black Pants Legion<br>
-	Musical backing is "Inspretional Wave" by khalafnasirs 2020 (CC-NO 3.0)"}
+	Musical backing is "Inspretional Wave" by khalafnasirs 2020 (CC-BY NO 3.0)"}
```
Comment (ZeWaka) [inline code/modules/telescience/radiostation.dm:632]:
```suggestion
	Musical backing is "Inspretional Wave" by khalafnasirs 2020 (CC-BY-NC 3.0)"}
```

File: code/modules/telescience/radiostation.dm [hunk 4]
```diff
@@ -626,7 +629,31 @@
...
+	
+/obj/item/radio_tape/advertisement/captain_psa
+	name = "compact tape - 'Captain's Training Program'"
+	audio = "sound/radio_station/Captain's Training.ogg"
+	name_of_thing = "Nanotrasen Captain's Training Promotional Tape"
+	desc = {"A small audio tape. Though, it looks to big to fit in an audio log.<br>
+	Voiceover by Tex of the Black Pants Legion<br>
+	Musical backing is "Out of Space" by Javolenus 2019 CC-BY NO 3.0"}
```
Comment (ZeWaka) [inline code/modules/telescience/radiostation.dm:640]:
```suggestion
	Musical backing is "Out of Space" by Javolenus 2019 CC-BY NC 3.0"}
```

File: code/__secret_public.dme (hunk 1 above)
Comment (PhuzzySA) [reply code/__secret_public.dme:25]: I didn't open it! I'm not sure how that got in there!

File: code/__secret_public.dme [hunk 5]
```diff
@@ -13,4 +13,4 @@
 #if SECRETS_ENABLED == 0
 #undef SECRETS_ENABLED
 #endif
-#endif
+#endif
```
Comment (ZeWaka) [inline code/__secret_public.dme:16]:
```suggestion
#endif

```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**